### DISCUSSION TITLE
Add the possibility to rename folders

### DISCUSSION THREAD

**Discussion by MrYamous** [opcodesio/log-viewer#407] [Ideas]
Hello 

I'm currently using log-viewer to see logs from several projets on the same server in the same place, using their absolute path in `include_files` configuration.
It works like a charm but as an improvement I wonder if it would be possible to rename folders because currently the paths are displayed (see screenshot). This is not ideal for readability.
![logs](https://github.com/user-attachments/assets/cca91d01-1d8a-47d7-8ff7-9d223207ef3e)

Ideally, I would like to be able to rename each folder (maybe not root)
Do you think this is a realistic feature?

**Comment by arukompas** [comment]
hey @MrYamous , great suggestion!

I've just tagged **v3.14.0** which now allows grouping and renaming these. You can get the gist of the config from the example here:
https://github.com/opcodesio/log-viewer/blob/876166da7af4864faca7100e63534116c1c295b9/config/log-viewer.php#L140-L148

This way you can group multiple inclusion patterns under one folder.

Let me know if you experience any issues!

**Reply by MrYamous** [reply]
Hello @arukompas
Thanks for this great news ! I will try it on monday and get back to you :)

**Reply by MrYamous** [reply]
I've just updated to 3.14 on my project, everything works fine
